modules/apps/commerce/commerce-product-content-api/src/main/java/com/liferay/commerce/product/content/helper/CPContentHelper.java

```diff
@@ -120,7 +120,7 @@ public String getFriendlyURL(
 		throws PortalException;
 
 	public List<CPMedia> getImages(
-			long cpDefinitionId, ThemeDisplay themeDisplay)
+			long cpDefinitionId, boolean gallery, ThemeDisplay themeDisplay)
```

Comment:
why not galleryEnabled as in other places?

Reply:
Because it doesn't refer to the field, but it refers to where it is called the function. I can change it if `galleryEnabled` suits better for you.